recipes/libcurl/all/conanfile.py

```diff
@@ -67,8 +67,8 @@ class LibcurlConan(ConanFile):
         "with_symbol_hiding": [True, False],
         "with_unix_sockets": [True, False],
         "with_verbose_strings": [True, False],
-        "with_ca_bundle": "ANY",
-        "with_ca_path": "ANY",
+        "with_ca_bundle": [None, "ANY"],
+        "with_ca_path": [None, "ANY"],
```

Comment:
You also need to set the default https://docs.conan.io/en/latest/migrating_to_2.0/recipes.html#any-special-value

Reply:
The default is None, that's the reason of this fix actually.